qunetsim/components/host.py

```diff
@@ -1341,6 +1341,12 @@ def add_data_qubit(self, host_id, qubit, q_id=None):
         Returns:
             (str): The qubit ID
         """
+        warnings.warn(
```

Comment:
I see the deprecation message, but. I don't see the new method called `add_qubit`. It seems to be missing... Do the tests pass with this code?